spikesorters/herdingspikes/herdingspikes.py

```diff
@@ -143,7 +143,7 @@ def _run(self, recording, output_folder):
 
     @staticmethod
     def get_result_from_folder(output_folder):
-        return se.HS2SortingExtractor(file_path=Path(output_folder) / 'HS2_sorted.hdf5')
+        return se.HS2SortingExtractor(file_path=Path(output_folder) / 'HS2_sorted.hdf5', load_unit_info=True)
```

Comment:
Why not change this default param directly in spikeextractors ?

Reply:
You're right, should do that!